src/server/worldserver/worldserver.conf.dist

```diff
@@ -812,6 +813,24 @@ HeroicCharactersPerRealm = 1
 CharacterCreating.MinLevelForHeroicCharacter = 55
 
 #
+#    DemonHuntersPerRealm
+#        Description: Limit number of demon hunter characters per account on this realm.
+#        Range:       1-12
+#        Default:     1
+
+DemonHuntersPerRealm = 1
+
+#
+#    CharacterCreating.MinLevelForDemonHunter
+#        Description: Limit creating demon hunters only for account with another
+#                     character of specific level.
+#        Default:     70 - (Enabled, Requires at least another level 70 character)
+#                     0  - (Disabled)
+#                     1  - (Enabled, Requires at least another level 1 character)
+
+CharacterCreating.MinLevelForDemonHunter = 70
```

Comment:
It is in the client and so I think that this is going to be true that there is a level requirement. If it's a false implementation it can be dropped easily. According to wowwiki and forums the level 70 cap is intended http://wowwiki.wikia.com/wiki/Demon_Hunter